bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/entity/AlternativePostalAddress.kt

```diff
@@ -98,4 +65,7 @@ class AlternativePostalAddress(
      */
     @Column(name = "alt_delivery_service_number")
     val deliveryServiceNumber: String = "",
+
+    @Column(name = "alt_delivery_service_qualifier")
+    val deliveryServiceQualifier: String = "",
```

Comment:
Since this is a new column added here, the existing business partners have NULL written in it. Since this entity expects a non null value it leads to an error. In general, if an entity field is not nullable, its column should be not nullable